src/app/api/auth/lib/nextauth-options.ts

```diff
@@ -87,7 +87,8 @@ export const refreshAccessToken = async (
       ),
     }
   } catch (error) {
-    authLogger.error('Error refreshing access token', { error })
+    authLogger.error(new Error('Error refreshing access token'))
```

Comment:
Anything actionable or more specific we can add to this statement? Is there information in the `error` object we caught that should be passed through to the logger?

Reply:
The logger gets the error stack without it having to be passed to the log function.  [Docs](https://betterstack.com/community/guides/logging/how-to-install-setup-and-use-winston-and-morgan-to-log-node-js-applications/#logging-errors-in-winston)

Reply:
`log.error()` only supports a single `Error` arg and it handles the error stack internally.

Reply:
Nice!